src/hydra/hydraClient.js

```diff
@@ -120,8 +121,10 @@ export default ({entrypoint, resources = []}, httpClient = fetchHydra) => {
    * @returns {Promise}
    */
   const convertReactAdminDataToHydraData = (resource, data = {}) => {
-    const fieldData = [];
-    resource.fields.forEach(({name, normalizeData}) => {
+    const {fields = []} = resources.find(({name}) => resource === name) || {};
```

Comment:
find is useless here:

```
const fieldData = [];
resource.fields.forEach(({name, normalizeData}) => {
```

Maybe that the assignation to `[]` is bc `resource.fields` can be undefined?